doc/distrib/NodeHelpFiles/_review/Dynamo.Nodes.DSEdgeSelection.md

```diff
@@ -1,7 +1,9 @@
 ## In Depth
-`Select Edge` brings up a cursor in Revit that allows you to select a single edge. The object returned is a curve or line that represents the selected edge.
+
+`Select Edge` displays a cursor in Revit that allows you to select a single edge. The object returned is a curve or line that represents the selected edge.
```

Comment:
Suggest adding for clarity - Additionally, `Select Edge` reports the owning element id of the edge that is selected.